myapp/api/v1/views/office/office.py

```diff
@@ -1,31 +1,28 @@
 from flask import Flask,Blueprint,jsonify,request,make_response
 import json
-from api.v1.models.Office.office import Office
+from v1.models.office.office import Office
 
 officeblue=Blueprint("/office",__name__)
 
 #add a new office
 @officeblue.route('/office', methods=['POST'] )
-def addoffice():
-    if request.json:
-        data=request.get_json()        
-        office_name=data['office_name']
-        office_type=data['office_type']
-        newoffice=Office(office_name,office_type)  
-        newoffice.addoffice()    
-        return make_response(jsonify(data,{"status":200,"message":"office successfully created"}))
-    else:
-        return make_response(jsonify({"status":404,"message":"office not created"}))
-
+def add_office():
+        if request.get_json():                
```

Comment:
Hi, are you requesting for user data twice?

Reply:
no on which module I check it out